code/game/machinery/doors/brigdoors.dm

```diff
@@ -63,6 +63,24 @@
 			stat |= BROKEN
 		update_icon()
 
+/obj/machinery/door_timer/proc/print_report()
+	var/logname = input(usr, "Name of the guilty?","[id] log name")
+	var/logcharges = input(usr, "What are they being charged with?","[id] log charges")

```

Comment:
Since they're already in the cell at this point, maybe it should be past tense of "What have they been changed with?"

Probably fine either way.